tracer/src/Datadog.Trace.ClrProfiler.Native/calltarget_tokens.cpp

```diff
@@ -884,7 +885,9 @@ HRESULT CallTargetTokens::WriteBeginMethod(void* rewriterWrapperPtr, mdTypeRef i
     ILRewriterWrapper* rewriterWrapper = (ILRewriterWrapper*) rewriterWrapperPtr;
     ModuleMetadata* module_metadata = GetMetadata();
 
-    auto numArguments = (int) methodArguments.size();
+    auto numArgsToLoad = argsToLoad.size();
+    bool useArgToLoad = numArgsToLoad > 0;
```

Comment:
Seems unlikely, but would we ever want to load `0` arguments (i.e. just access the instance)?

Reply:
It's possible, so I added another field to allow us to disambiguate between `0` arguments and NOT use the feature in f9b0cf01e11c2487c2f6075a738c919587022ecb